destruct/create_ref_data.py

```diff
@@ -45,45 +45,46 @@ def create_ref_data(config, ref_data_dir):
     except OSError:
         pass
 
-    def wget_genome_fasta():
-        with open(config['genome_fasta'], 'w') as genome_file:
-            for assembly in config['ensembl_assemblies']:
-                assembly_url = config['ensembl_assembly_url'].format(assembly)
-                assembly_fasta = os.path.join(temp_directory, 'dna.assembly.{0}.fa'.format(assembly))
-                if not os.path.exists(assembly_fasta):
-                    wget_gunzip(assembly_url, assembly_fasta)
-                with open(assembly_fasta, 'r') as assembly_file:
-                    for line in assembly_file:
-                        if line[0] == '>':
-                            line = line.split()[0] + '\n'
-                        genome_file.write(line)
-    auto_sentinal.run(wget_genome_fasta)
-
-    def wget_gtf():
-        wget_gunzip(config['ensembl_gtf_url'], config['gtf_filename'])
-    auto_sentinal.run(wget_gtf)
-
-    def wget_dgv():
-        wget(config['dgv_url'], config['dgv_filename'])
-    auto_sentinal.run(wget_dgv)
-
-    def wget_repeats():
-        repeat_filename = os.path.join(temp_directory, 'repeats.txt')
-        wget_gunzip(config['rmsk_url'], repeat_filename)
-        with open(config['chromosome_map'], 'r') as chr_map_file:
-            chr_map = dict((a.split() for a in chr_map_file))
-        with open(repeat_filename, 'r') as repeat_file, open(config['repeat_regions'], 'w') as repeat_regions_file, open(config['satellite_regions'], 'w') as satellite_regions_file:
-            for row in csv.reader(repeat_file, delimiter='\t'):
-                if row[5] not in chr_map:
-                    continue
-                chr = chr_map[row[5]]
-                start = str(int(row[6]) + 1)
-                end = row[7]
-                type = row[11]
-                repeat_regions_file.write('\t'.join([chr, start, end]) + '\n')
-                if type == 'Satellite':
-                    satellite_regions_file.write('\t'.join([chr, start, end]) + '\n')
-    auto_sentinal.run(wget_repeats)
+   def wget_genome_fasta():
+       print (config["genome_fasta"])
+       with open(config['genome_fasta'], 'w') as genome_file:
```

Comment:
also dont see any changes. can you make sure that git doesnt track this section as changed between commits.